### PR TITLE
feat(query): added "Glue Security Configuration Encryption Disabled" for Terraform

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- Added "Glue Security Configuration Encryption Disabled" for Terraform
- **Category and Severity are provisory**

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+	configValue := configs[configKey]
+	not common_lib.valid_key(encryptionConfig[configKey], configValue)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_glue_security_configuration[%s].%s", [name, configKey]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("aws_glue_security_configuration[%s].%s has '%s' defined and not null", [name, configKey, configValue]),
+		"keyActualValue": sprintf("aws_glue_security_configKeyiguration[%s].%s has '%s' undefined or null", [name, configKey, configValue]),
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/query.rego:24]: missing searchLine

File: assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/query.rego [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+	config := configs[c]
+	not common_lib.valid_key(resource.encryption_configuration[config], "kms_key_arn")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_glue_security_configuration[%s].%s", [name, config]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("aws_glue_security_configuration[%s].%s has 'kms_key_arn' defined and not null", [name, config]),
+		"keyActualValue": sprintf("aws_glue_security_configuration[%s].%s has 'kms_key_arn' undefined or null", [name, config]),
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/query.rego:44]: missing searchLine

File: assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	resource := input.document[i].resource.aws_glue_security_configuration[name]
+
+	searchKeyInfo := wrong_config(resource.encryption_configuration)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_glue_security_configuration[%s].%s", [name, searchKeyInfo.path]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": searchKeyInfo.keyExpectedValue,
+		"keyActualValue": searchKeyInfo.keyActualValue,
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/query.rego:58]: missing searchLine

File: assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/metadata.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ad5b4e97-2850-4adf-be17-1d293e0b85ee",
+  "queryName": "Glue Security Configuration Encryption Disabled",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "Glue Security Configuration Encryption should have 'cloudwatch_encryption', 'job_bookmarks_encryption' and 's3_encryption' enabled",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/glue_resource_policy#policy",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/metadata.json:7]:
```suggestion
  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/glue_security_configuration",
```

File: assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/metadata.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ad5b4e97-2850-4adf-be17-1d293e0b85ee",
+  "queryName": "Glue Security Configuration Encryption Disabled",
+  "severity": "MEDIUM",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/metadata.json:4]:
```suggestion
  "severity": "HIGH",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/metadata.json (hunk 5 above)
Comment (JoaoDuarteGomes) [reply assets/queries/terraform/aws/glue_security_configuration_encryption_disabled/metadata.json:4]: I agree with this suggestion.

**Review by JoaoDuarteGomes — COMMENTED** [review]
I agree with the category selection and with the change suggested by @rogeriopeixotocx for the severity.

**Review by rogeriopeixotocx — APPROVED** [review]

**Pull request merged**